### PR TITLE
Add CRUD permission for the api token

### PR DESCRIPTION
**Pull request by Elhebert**
### What does it do?

This PR introduce the permissions for the API token.

### Why is it needed?

Registering the permission is the first step in adding the API token feature into the admin.

### How to test it?

The code is supported by the e2e test: Role CRUD End to End - Can get the existing permissions (`packages/core/admin/server/tests/admin-permission.test.e2e.js`)

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### CODE REVIEW

**Review by alexandrebodin — APPROVED** [review]
LGTM

File: packages/core/admin/server/config/admin-actions.js [hunk 1]
```diff
@@ -118,5 +118,33 @@ module.exports = {
       category: 'users and roles',
       subCategory: 'roles',
     },
+    {
+      uid: 'api-tokens.create',
+      displayName: 'Create',
```
Comment (petersg83) [inline packages/core/admin/server/config/admin-actions.js:123]: Create (generate)

File: packages/core/admin/server/config/admin-actions.js [hunk 2]
```diff
@@ -118,5 +118,33 @@ module.exports = {
...
+    {
+      uid: 'api-tokens.update',
+      displayName: 'Update',
+      pluginName: 'admin',
+      section: 'settings',
+      category: 'api tokens',
+    },
+    {
+      uid: 'api-tokens.delete',
+      displayName: 'Delete',
```
Comment (petersg83) [inline packages/core/admin/server/config/admin-actions.js:144]: Delete (revoke)

**Review by petersg83 — CHANGES_REQUESTED** [review]
Is almost good :) The display names are maybe a bit different. To check with @JabStrapi

**Review by petersg83 — APPROVED** [review]
LGTM too !
Well done for this first PR :D 🎉 🌮

**Pull request merged**